mods/minimal/utility.lua

```diff
@@ -26,7 +26,7 @@ function minimal.safe_landing_spot(pos)
 				      def_top.walkable == true ) then
       return false -- loaded a solid node
    end
-   if dest_bot.name ~= "ignore" and ( def_bot and
+   if dest_bot.name ~= "ignore" and ( _bot and
```

Comment:
Looks like an accidental change here

Reply:
hmm, not sure how that happened
I'll remove it